### PR TITLE
For #76: Exec register error tests.

### PR DESCRIPTION
**Pull request by paulodamaso**
For #76:
- Created test for `Exec` error register;
- Implemented simple version of `Error` and `Errors`
- Added constructor to `Exec` to receive error repository reference
- Updated todo to continue implementation

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#79](https://github.com/yegor256/wring/issues/79) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#79](https://github.com/yegor256/wring/issues/79) is assigned to @pinaf[/z](https://www.0crat.com/u/pinaf), here is [why](https://www.0crat.com/footprint/C7FCB1EQN/6eb48b82-637b-4ce8-b15f-3a12d448de1f); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by paulodamaso** [comment]
@pinaf ping

File: src/main/java/io/wring/model/Error.java [hunk 1]
```diff
@@ -53,4 +53,40 @@
...
+         * @param description Description
+         */
+        public Simple(final String title, final String description) {
+            this.title = title;
+            this.description = description;
+        }
+
+        @Override
+        public Iterable<Directive> asXembly() throws IOException {
+            throw new UnsupportedOperationException("not implemented");
```
Comment (pinaf) [inline src/main/java/io/wring/model/Error.java]: @paulodamaso let's add a puzzle for the implementation of this method

File: src/main/java/io/wring/model/Error.java [hunk 2]
```diff
@@ -53,4 +53,40 @@
...
+        }
+
+        @Override
+        public void delete() throws IOException {
+            throw new UnsupportedOperationException("delete() not implemented");
```
Comment (pinaf) [inline src/main/java/io/wring/model/Error.java]: @paulodamaso let's add a puzzle for the implementation of this method

**Comment by paulodamaso** [comment]
@pinaf Please take a look.

**Comment by pinaf** [comment]
@paulodamaso thanks
@carlosmiranda good to merge

**Comment by paulodamaso** [comment]
@carlosmiranda ping

**Comment by carlosmiranda** [comment]
@rultor merge

File: src/main/java/io/wring/model/Errors.java [hunk 3]
```diff
@@ -57,4 +56,32 @@
...
+
+        /**
+         * Constructor.
+         */
+        public Simple() {
+            this.errors = new ArrayList<>(0);
+        }
+
+        @Override
+        public Iterable<Error> iterate() throws IOException {
```
Comment (pinaf) [inline src/main/java/io/wring/model/Errors.java]: @paulodamaso let's remove the exception here

**Comment by paulodamaso** [comment]
@pinaf I think that it's good now

**Comment by pinaf** [comment]
@paulodamaso yep, thanks!
@carlosmiranda let's try to merge again.

**Comment by carlosmiranda** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @pinaf[/z](https://www.0crat.com/u/pinaf), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#79](https://github.com/yegor256/wring/issues/79) is now out of scope

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@4a62f1a2-e6e2-417c-9751-47fc0ca91c06, thread: AsyncProc-7
com.zerocracy.farm.reactive.StkRuntime[76] java.lang.IllegalStateException: org.codehaus.groovy.runtime.InvokerInvocationException in com.zerocracy.stk.pm.cost.funding.send_zold
org.codehaus.groovy.reflection.CachedMethod[105] org.codehaus.groovy.runtime.InvokerInvocationException: java.io.IOException: Zold payment failed, code=503
com.zerocracy.pmo.banks.Zold[83] java.io.IOException: Zold payment failed, code=503
```

[0.37.5](https://github.com/zerocracy/farm): CID: [947424ae-7e84-4d69-8404-d4fc2a40fbad](https://www.0crat.com/footprint/C7FCB1EQN/947424ae-7e84-4d69-8404-d4fc2a40fbad), Type: "Send zold"

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @pinaf[/z](https://www.0crat.com/u/pinaf)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**